docs/contributing/docs_style_guide.md

```diff
@@ -10,9 +10,20 @@
 - Use `-` instead of `*` for bulleted lists.
 - Upload images to the 'assets' folder and reference them from there. 
    -- For file name, use underscores between words and prefix all file names with the page name, e.g. context_display_hints.jpg for the image showing how to set display hints in the context menu.
-- Use `!!! note  "Note title"` at the start of a paragraph to have it rendered as a note like this:
+- Use the [Admonition syntax](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) to create notes like this:
+
+_Example:_
+
+```
+!!! note "Helpful Tip" 
+  I am a helpful tip!
```

Comment:
Looks like this needs 5 spaces to be indented enough to stay in the note box.
```suggestion
    I am a helpful tip!
```

Reply:
yeah, the docs I included a link to explicitly say four. Sorry. Also! I've put that right in the docs because I'm sure I'm not the only one to mess that up; feel free to suggest to take out.

Reply:
Oops, clearly my counting is not up to snuff. 4 spaces it is. 👍